src/Tasks/Common/Resources/Strings.resx

```diff
@@ -694,4 +694,8 @@ The following are names of parameters or literal values and should not be transl
     <value>NETSDK1147: The following workload packs were not installed: {0}</value>
     <comment>{StrBegin="NETSDK1147: "}</comment>
   </data>
+  <data name="WindowsSDKVersionConflicts" xml:space="preserve">
+    <value>NETSDK1148: Version conflicts were found for Microsoft.Windows.SDK.NET.dll. Please update the .NET SDK to resolve this conflict.</value>
```

Comment:
Here's my suggestion for improving the wording:

```suggestion
    <value>NETSDK1148: A referenced assembly was compiled using a newer version of Microsoft.Windows.SDK.NET.dll. Please update to a newer .NET SDK in order to reference this assembly..</value>
```

@KathleenDollard @AdamBraden @scottj1s What do you think of the wording?

@Forgind @BenVillalobos Is it always going to be accurate that a version conflict would be caused by referencing an assembly that references a newer version?  Or is there some other way that ResolveAssemblyReference could find a version conflict?

@wli3 We're past the Loc date for 15.9, right?  Do we need to get an exception for this?

Reply:
I don't remember all the details of how RAR chooses the conflict victor, but I think that's always true if it logs a warning and never true if it just logs a message.

Reply:
I've got very little experience with RAR, so I'll defer to Nathan on this one.

Reply:
Updated wording LGTM